site/edit/template/layout.jsx

```
@@ -16,169 +14,165 @@ import Iframe from './components/Iframe';
+    
+    if (templateData.data.user && templateData.data.user.userId && 
+        !templateData.data.user.delete && !userIsLogin) {
       return (
-        <div className="login-controller" key="1">
-          <div className={`login-view${passwordNo ? ' password-no' : ''}`}>
-            <a href={getNewHref('7111', null, true)} target="_blank" className="header">
+        <div className="flex items-center justify-center min-h-screen bg-gray-100" key="1">
+          <div className={`bg-white p-8 rounded-lg shadow-md w-full max-w-md ${
+            this.state.error ? 'border border-red-500' : ''
+          }`}>
+            <a 
```

ESLint

[error] Trailing spaces not allowed.

(no-trailing-spaces)